### PR TITLE
Updated EDA with new plot on class imbalance

### PR DESCRIPTION
**Pull request by sean-m-mckay**
- Updated README with instructions to run tests
- Update gitignore to ignore build files

### CODE REVIEW

File: report/wine_classification_report.ipynb [hunk 1]
```diff
@@ -17,7 +17,19 @@
      "hide-cell"
     ]
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "ename": "ModuleNotFoundError",
+     "evalue": "No module named 'sklearn'",
+     "output_type": "error",
```
Comment (juliaeveritt) [inline report/wine_classification_report.ipynb:24]: Hmmm - are these errors expected?

Comment (sean-m-mckay) [reply report/wine_classification_report.ipynb:24]: Good catch, let me take another look!

Comment (sean-m-mckay) [reply report/wine_classification_report.ipynb]: Updated!

**Review by hbandukw — APPROVED** [review]
LGTM!

**Pull request merged**